content/yaml-code-signing/code-signing-identities.md

```diff
@@ -0,0 +1,140 @@
+---
+title: Code signing identities
+description: How to set up code signing identities
+weight: 4
+aliases: /code-signing-yaml/code-signing-identities
+---
+
+Teams can use code signing identities to manage their code signing files in **Team settings > codemagic.yaml settings > Code signing identities** and reference them from their `codemagic.yaml` configuration.
+
+## Managing and uploading files
+
+Team owner permissions are required to upload and edit files under the **Code signing identities** section. However, all team members can view the file info for any of the uploaded files.
+
+### Apple certificates
+
+Codemagic lets you upload certificates with the `.p12` or `.pem` extension. When uploading, Codemagic will ask you to provide the certificate password along with a unique **Reference name**, which can then be used in the `codemagic.yaml` configuration to fetch the specific file.
+
+In addition, Codemagic provides the option to generate a new `Development` or `Distribution` certificate. Click on **Generate certificate**, provide a **Reference name** and choose the certificate type and a matching bundle identifier. Once the certificate has been created, Codemagic will allow you to download the certificate and provides the password for it. The download is available only once.
```

Comment:
Apple Development or Apple Distribution certificate.
Not bundle ID but API key.

Reply:
Also make the note about download more prominent: 
Note that the certificate can be downloaded just once right after creating it.